index.html

```diff
@@ -0,0 +1,118 @@
+    <link rel="stylesheet" href="assets/css/style.css">
+    <!-- icons -->
+    <link href='https://cdn.jsdelivr.net/npm/boxicons@2.0.5/css/boxicons.min.css' rel='stylesheet'>
+</head>
+<body>
+    <header class="nav">
+        <nav id="navbar">
+            <div>
+                <a href="#" class="nav__logo">Microverse</a>
+            </div>
+
+            <div class="nav_menu">
+                <ul class="nav__list">
+                    <li class="nav__item" > <a href="">Home</a> </li>
+                    <li class="nav__item" > <a href="">Projects</a> </li>
+                    <li class="nav__item"> <a href="">User Stories</a> </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+
+    <main class="l-main">
+        <section id="welcome-section">
+            <h1>
+                Hey I am mimic
+            </h1>
+            <p>
+                a web developer
+            </p>
+        </section>
+
+        <section class="work" id="projects">
+            <h2 class="section-title">These are some of my projects</h2>
+
+                <!-- here is the list of different projects -->
+                <div class="work__container bd-grid">
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work1.jpg" alt="">
+                    </div>
+
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work2.jpg" alt="">
+                    </div>
+
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work3.jpg" alt="">
+                    </div>
+
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work4.jpg" alt="">
+                    </div>
+
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work5.jpg" alt="">
+                    </div>
+
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" class="project_title">Project Title</a>
+                        <img src="assets/img/work6.jpg" alt="">
+                    </div>
+                </div>
+            
+        </section>
+        <!-- <section id="user-strories">
+            User stories
+        </section> -->
```

Comment:
I think it's a nice practice to delete commented codes from your project to have a clean project